### ISSUE TITLE
There is no user feedback while the live photo is loading

### ISSUE THREAD

**Issue by idvorkin** [tumblr/laphs#7]
Would be cool if there was feedback while the movie is downloading, otherwise seems like the live photo is stalled.

**Comment by heylookltsme** [comment]
Agree! That was on our radar but didn't make it into the library before we needed to ship it. It would definitely be a great improvement. Pull requests welcome! ;)
